tuxemon/technique.py

```diff
@@ -239,9 +254,21 @@ def use(self, user: Monster, target: Monster) -> TechniqueResult:
             "capture": False,
             "statuses": [],
         }
-
         # TODO: handle conflicting values from multiple technique actions
         # TODO: for example, how to handle one saying success, and another not?
+        statsmaster = [self.statspeed, self.stathp, self.statarmour, self.statmelee, self.statranged, self.statdodge]
+        statslugs = ['speed', 'hp', 'armour', 'melee', 'ranged', 'dodge']
+        print(statsmaster)
+        if None not in statsmaster:
+            for stat in statsmaster:
+                for paramater in range(len(stat)):
+                    print(stat)
+                    value = stat['value']
+                    dividing = stat['dividing']
+                    override = stat['overridetofull']
+                    if value > 0 and override == False:
+                        statvalue = getattr(target, statslugs[paramater])
+                        print(statvalue)
```

Comment:
I am not sure what do you want to do here, but probably you want to loop for similar things together (e.g. `self.statarmour` and `"armour"`), right?
```suggestion
            for stat, slug in zip(statsmaster, statslugs):
                print(stat)
                value = stat['value']
                dividing = stat['dividing']
                override = stat['overridetofull']
                if value > 0 and override == False:
                    statvalue = getattr(target, slug)
                    print(statvalue)
```